website/screens/components/paginator/code/PaginatorCodePage.tsx

```diff
@@ -19,76 +19,101 @@ const sections = [
         <thead>
           <tr>
             <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
             <th>Default</th>
-            <HeaderDescriptionCell>Description</HeaderDescriptionCell>
           </tr>
         </thead>
         <tbody>
           <tr>
-            <td>currentPage: number</td>
+            <td>currentPage</td>
             <td>
-              <Code>1</Code>
+              <TableCode>number</TableCode>
             </td>
             <td>Number of the current selected page.</td>
+            <td>
+              <TableCode>1</TableCode>
+            </td>
           </tr>
           <tr>
-            <td>itemsPerPage: number</td>
+            <td>itemsPerPage</td>
             <td>
-              <Code>5</Code>
+              <TableCode>number</TableCode>
             </td>
             <td>Number of items per page.</td>
+            <td>
+              <TableCode>5</TableCode>
+            </td>
           </tr>
           <tr>
-            <td>itemsPerPageOptions: number[]</td>
+            <td>itemsPerPageOptions</td>
             <td>
-              <Code>[]</Code>
+              <TableCode>number[]</TableCode>
             </td>
             <td>
               An array of numbers representing the items per page options. If
               undefined, the select with items per page options will not be
               displayed.
             </td>
+            <td>-</td>
           </tr>
           <tr>
-            <td>itemsPerPageFunction: function</td>
-            <td></td>
+            <td>itemsPerPageFunction</td>
+            <td>
+              <TableCode>{"(itemsPerPage: number) => void"}</TableCode>
+            </td>
             <td>
               This function will be called when the user selects an item per
               page option.
```

Comment:
I would add an explanation about what is being passed as parameter the same as onPageChange prop.

Reply:
Agree! I will update the description.